docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -309,6 +312,9 @@ The hexadecimal ("X") format specifier converts a number to a string of hexadeci
 
 The precision specifier indicates the minimum number of digits desired in the resulting string. If required, the number is padded with zeros to its left to produce the number of digits given by the precision specifier.
 
+For <xref:System.Numerics.BigInteger>, positive values always have a leading zero to distinguish them from negative values. This ensures the output round-trips to the original value when parsed.
```

Comment:
It might be worth adding a similar note to `NumberStyles.BinaryNumber/HexNumber` for parsing.